cmd/configschema/go.mod

```diff
@@ -344,7 +344,7 @@ require (
 	github.com/santhosh-tekuri/jsonschema v1.2.4 // indirect
 	github.com/scaleway/scaleway-sdk-go v1.0.0-beta.7.0.20210223165440-c65ae3540d44 // indirect
 	github.com/shirou/gopsutil v3.21.8+incompatible // indirect
-	github.com/shirou/gopsutil/v3 v3.21.6 // indirect
+	github.com/shirou/gopsutil/v3 v3.21.9 // indirect
```

Comment:
These are "indirect" deps, if I merge this, download the code locally, re-run go mod I think they will get reverted correct? If you want to enforce this probably a replace statement is better or find who uses this as direct dep and fix there.

Reply:
Since they were already there, they will be kept there by `go mod tidy`